src/main/java/seedu/address/logic/LogicTrackermon.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic;
+
+import java.nio.file.Path;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.logic.commands.CommandResultTrackermon;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.ReadOnlyShowList;
+import seedu.address.model.show.Show;
+
+/**
+ * API of the Logic component
+ */
+public interface LogicTrackermon {
+    /**
+     * Executes the command and returns the result.
+     * @param commandText The command as entered by the user.
+     * @return the result of the command execution.
+     * @throws CommandException If an error occurs during command execution.
+     * @throws ParseException If an error occurs during parsing.
+     */
+    CommandResultTrackermon execute(String commandText) throws CommandException, ParseException;
+
+    /**
+     * Returns the ShowList.
+     *
+     * @see seedu.address.model.ModelTrackermon#getShowList()
+     */
+    ReadOnlyShowList getShowList();
+
+    /** Returns an unmodifiable view of the filtered list of shows */
+    ObservableList<Show> getFilteredShowList();
+
+    /**
+     * Returns the user prefs' address book file path.
```

Comment:
Should this be "Returns the user prefs' show list file path" instead?

Reply:
Updated